chained-actions/index.json

```diff
@@ -0,0 +1,64 @@
+{
+    "pathwayTitle": "Apache OpenWhisk on OpenShift",
+    "icon": "fa-openshift",
+    "title": "Chained Actions",
+    "description": "Chaining Actions on OpenWhisk",
+    "difficulty": "intermediate",
+    "time": "15 minutes",
+    "details": {
+        "intro": {
+            "text": "00-intro.md",
+            "courseData": "env-init.sh",
+            "credits": "http://developers.redhat.com",
+            "code": "set-env.sh"
+        },
+        "steps": [
+            {
+                "title": "Step 1",
+                "text": "create-package.md"
+            },
+            {
+                "title": "Step 2",
+                "text": "create-split-function.md"
+            },
+            {
+                "title": "Step 3",
+                "text": "create-sort-function.md"
+            },
+            {
+                "title": "Step 3",
+                "text": "create-uppercase-function.md"
+            },
+            {
+                "title": "Step 3",
+                "text": "create-sequence.md"
+            }
+        ],
+        "assets": {
+            "host01": [
+                {
+                    "file": "configure-owsk.sh",
+                    "target": "/tmp/"
+                }
+            ]
+        }
+    },
+    "files": [],
+    "environment": {
+        "showdashboard": true,
+        "dashboards": [
+            {
```

Comment:
dashboards should be empty with [].  This scenario does not use the OpenShift Console.